pkg/imgutil/nativeimgutil/nativeimgutil.go

```diff
@@ -38,10 +42,17 @@ func roundUp(size int64) int64 {
 	return sectors * sectorSize
 }
 
-// convertToRaw converts a source disk into a raw disk.
+type targetImageType string
+
+const (
+	imageRaw  targetImageType = "raw"
+	imageASIF targetImageType = "ASIF"
```

Comment:
lower chars and upper chars should be used consistently

Reply:
https://pkg.go.dev/github.com/lima-vm/go-qcow2reader@v0.7.1/image/asif#pkg-constants should suffice anyway